init.zsh

```diff
@@ -17,8 +17,13 @@
   # Move cursor to end of word if a full completion is inserted.
   setopt ALWAYS_TO_END
 
-  # Make globbing case insensitive.
-  setopt NO_CASE_GLOB
+  if zstyle -t ':zim:completion' case-sensitive; then
```

Comment:
I've been considering having this option, and I think:
* it should have a more global name (not module-specific), since many modules can use the same option to configure themselves regarding case sensitivity.
* value should allow for more than `yes`/`no` as option, so it could allow for `sensitive`, `insensitive` and `smart`, for example.

Reply:
smart being to be insensitive if the provided case is not present? Sounds like a more comprehensive solution. What to call the option...

Reply:
Start a 'zim:global` namespace or something? 

`zstyle 'zim:global' case-sensitivity smart`